static-analysis/teammates-checkstyle.xml

```diff
@@ -173,6 +173,27 @@
       <property name="onCommentFormat" value="CHECKSTYLE\.ON:([A-Za-z\|]+)"/>
       <property name="checkFormat" value="$1"/>
     </module>
+    <module name="FallThrough"/>
+    <module name="ArrayTrailingComma"/>
+    <module name="IllegalThrows">
+      <property name="ignoredMethodNames" value="execute"/>
+      <!-- Remove execute value from this property after upgrading to JUnit 5 (see teammates.test.cases.Executable) -->
+    </module>
+    <module name="InterfaceIsType"/>
+    <module name="MutableException"/>
+    <module name="OverloadMethodsDeclarationOrder"/>
+    <module name="SummaryJavadocCheck"/>
+    <module name="JavadocStyle">
+      <property name="scope" value="package"/>
+      <property name="checkEmptyJavadoc" value="true"/>
+    </module>
+    <module name="ImportOrder">
```

Comment:
There are already configurations for this and `JavadocStyle`. Let's not be hasty.